src/constants/translations/en/index.js

```diff
@@ -23,7 +23,7 @@ import completeProfile from './complete-profile.json'
 import tutorProfilePage from './tutor-profile-page.json'
 import adminTable from './admin-table.json'
 import constant from './constant.json'
-import findOffers from './find-offer-page.json'
+import findOffer from './find-offer-page.json'
```

Comment:
why do you rename it? if this page is find-offers across the app, if we need to rename we should rename all files

Reply:
You are right. I attached the screen where needs to fix his problem)